src/app/containers/Brand/index.jsx

```diff
@@ -1,22 +1,59 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
+import SkipLink from '@bbc/psammead-brand/skip-link';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { bool } from 'prop-types';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const renderScriptLink = (
+  script,
+  service,
+  variant,
+  url,
+  scriptLinkText,
+  offscreenText,
+) => (
+  <ScriptLink script={script} service={service} href={url} variant={variant}>
```

Comment:
The service variant preference cookie will be set when a user clicks on the ScriptLink, so we have to pass the `onClick` handler to it as a prop.

Reply:
Added the onClick handler